standalone/src/models/discount-group/discount-group-draft/types.ts

```diff
@@ -0,0 +1,10 @@
+import { DiscountGroupDraft } from '@commercetools/platform-sdk';
```

Comment:
We should remove this file as the draft model types already exist in the parent folder `types.ts` file.